griptape/drivers/prompt/openai_chat_prompt_driver.py

```diff
@@ -89,13 +120,35 @@ def try_stream(self, prompt_stack: PromptStack) -> Iterator[TextArtifact]:
         for chunk in result:
             if len(chunk.choices) == 1:
                 delta = chunk.choices[0].delta
-            else:
-                raise Exception("Completion with more than one choice is not supported yet.")
+                tool_call_deltas = delta.tool_calls
+
+                if tool_call_deltas:
+                    if len(tool_call_deltas) == 1:
+                        tool_call_delta = tool_call_deltas[0]
+
+                        if tool_call_delta.id is not None:
+                            name, path = tool_call_delta.function.name.split("-")
```

Comment:
Did you mean to check `tool_call_delta.function.name` (rather than `tool_call_delta.id`)?